docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[cases-open-manage]]
+= Open and manage cases
+
+[float]
+[[cases-ui-open]]
+== Open a new case
+
+Open a new case to keep track of security issues and share their details with
+colleagues.
+
+. Go to *Investigate* -> *Cases*, then click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table.
+. Give the case a name, add relevant tags, assign a severity, and provide a description.
++
+TIP: In the `Description` area, you can use
+https://www.markdownguide.org/cheat-sheet[Markdown] syntax.
+
+. Choose whether you want alert statuses to sync with the case's status after they are added to the case. This option is enabled by default, but you can still turn it off after creating the case.
+. From *External incident management*, select a <<cases-external-connectors,connector>>. If you’ve previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
```

Comment:
```suggestion
. From *External incident management*, select a <<cases-ui-integrations, connector>>. If you’ve previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
```
```suggestion
. From *External incident management*, select a <<cases-external-connectors,connector>>. If you’ve previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
```

Reply:
Resolving because duplicate edit.